src/main/java/domain/movement/Direction.java

```diff
@@ -0,0 +1,59 @@
+package domain.movement;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum Direction {
+    UP(1, 0),
+    DOWN(-1, 0),
+    LEFT(0, -1),
+    RIGHT(0, 1),
+    UP_LEFT(1, -1),
+    UP_RIGHT(1, 1),
+    DOWN_LEFT(-1, -1),
+    DOWN_RIGHT(-1, 1),
+    KNIGHT_UP_LEFT(2, -1),
+    KNIGHT_UP_RIGHT(2, 1),
+    KNIGHT_RIGHT_UP(1, 2),
+    KNIGHT_RIGHT_DOWN(-1, 2),
+    KNIGHT_DOWN_RIGHT(-2, 1),
+    KNIGHT_DOWN_LEFT(-2, -1),
+    KNIGHT_LEFT_DOWN(-1, -2),
+    KNIGHT_LEFT_UP(1, -2);
+
+    private static final List<Direction> KNIGHT = List.of(KNIGHT_UP_LEFT, KNIGHT_UP_RIGHT, KNIGHT_RIGHT_UP,
+            KNIGHT_RIGHT_DOWN,
+            KNIGHT_DOWN_RIGHT, KNIGHT_DOWN_LEFT, KNIGHT_LEFT_DOWN, KNIGHT_LEFT_UP);
+    private final int rankDiff;
```

Comment:
상수와 필드를 분리해 가독성을 챙겨봅시다
```suggestion
            KNIGHT_DOWN_RIGHT, KNIGHT_DOWN_LEFT, KNIGHT_LEFT_DOWN, KNIGHT_LEFT_UP);
    
    private final int rankDiff;
```